apm-sniffer/apm-sdk-plugin/jedis-plugins/jedis-2.x-3.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/jedis/v3/define/JedisInstrumentation.java

```diff
@@ -48,63 +38,17 @@ public ClassMatch enhanceClass() {
 
     @Override
     public ConstructorInterceptPoint[] getConstructorsInterceptPoints() {
-        return new ConstructorInterceptPoint[] {
-                new ConstructorInterceptPoint() {
-                    @Override
-                    public ElementMatcher<MethodDescription> getConstructorMatcher() {
-                        return takesArgument(0, String.class);
-                    }
-
-                    @Override
-                    public String getConstructorInterceptor() {
-                        return CONSTRUCTOR_WITH_STRING_ARG_INTERCEPT_CLASS;
-                    }
-                },
-                new ConstructorInterceptPoint() {
-                    @Override
-                    public ElementMatcher<MethodDescription> getConstructorMatcher() {
-                        return takesArgumentWithType(0, HOST_AND_PORT_ARG_TYPE_NAME);
-                    }
-
-                    @Override
-                    public String getConstructorInterceptor() {
-                        return CONSTRUCTOR_WITH_HOST_AND_PORT_INTERCEPT_CLASS;
-                    }
-                },
-                new ConstructorInterceptPoint() {
-                    @Override
-                    public ElementMatcher<MethodDescription> getConstructorMatcher() {
-                        return takesArgumentWithType(0, JEDIS_SHARD_INFO_ARG_TYPE_NAME);
-                    }
-
-                    @Override
-                    public String getConstructorInterceptor() {
-                        return CONSTRUCTOR_WITH_SHARD_INFO_ARG_INTERCEPT_CLASS;
-                    }
-                },
-                new ConstructorInterceptPoint() {
-                    @Override
-                    public ElementMatcher<MethodDescription> getConstructorMatcher() {
-                        return takesArgument(0, URI.class);
-                    }
-
-                    @Override
-                    public String getConstructorInterceptor() {
-                        return CONSTRUCTOR_WITH_URI_ARG_INTERCEPT_CLASS;
-                    }
-                }
-        };
+        return new ConstructorInterceptPoint[0];
     }
 
     @Override
     public InstanceMethodsInterceptPoint[] getInstanceMethodsInterceptPoints() {
-        return new InstanceMethodsInterceptPoint[] {
+        return new InstanceMethodsInterceptPoint[]{
```

Comment:
You seem not checking the style?

Reply:
maven build can success, i have fixed it

Reply:
I think this only fails in verify stage.